deps/lame.pc

```diff
@@ -0,0 +1,9 @@
+prefix=/app
```

Comment:
Did you write this pkgconfig file? LAME does not have one upstream.

Reply:
No, I got it from https://github.com/flathub/org.audacityteam.Audacity/blob/master/deps/lame.pc/

Reply:
Could you send this upstream to LAME? Unfortunately that requires using SourceForge :(

Reply:
Sadly I can't, because I have no idea how to use SourceForge.

Reply:
Reasonable. I don't want spam inserted into my emails either.

Reply:
it's more complicated. Blame "conan". It's the only reason a .pc is needed.